src/session/content/chat_history.rs

```diff
@@ -198,5 +200,11 @@ impl ChatHistory {
 
         let adj = self_.list_view.vadjustment().unwrap();
         self.load_older_messages(&adj);
+        if let Some(chat) = self.chat() {
```

Comment:
Why don't you put this in the previous `if let` statement?

Reply:
I do remember now. Initially there was no `if let` statement and then I just added if let when I was reverting some changes